unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -66,7 +66,7 @@ showPatternHelp i =
               then " (or " <> intercalate ", " (I.aliases i) <> ")"
               else ""
           ),
-      P.wrap $ I.help i
+      I.help i
```

Comment:
OMG, I can't believe it took this long to track this down. All `help` was being wrapped, which messes up formatting. As I was developing the `help` output for this, I found and fixed this.

With this change, we rely on each command to `wrap` if needed.